RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -105,17 +105,18 @@
                      'lowPtQuadStepTracks',
                      'lowPtTripletStepTracks',
                      'detachedQuadStepTracks',
-                     'pixelPairStepTracks'],
-    hasSelector = [1,1,1,1,1,1],
+#                     'pixelPairStepTracks'
+                    ],
+    hasSelector = [1,1,1,1,1],
     indivShareFrac = [1.0,0.16,0.095,0.09,0.09,0.09],
     selectedTrackQuals = cms.VInputTag(cms.InputTag("initialStepSelector","initialStep"),
                                        cms.InputTag("highPtTripletStepSelector","highPtTripletStep"),
                                        cms.InputTag("lowPtQuadStepSelector","lowPtQuadStep"),
                                        cms.InputTag("lowPtTripletStepSelector","lowPtTripletStep"),
                                        cms.InputTag("detachedQuadStep"),
-                                       cms.InputTag("pixelPairStepSelector","pixelPairStep")
+#                                       cms.InputTag("pixelPairStepSelector","pixelPairStep")
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant